packages/models-library/src/models_library/api_schemas_long_running_tasks/base.py

```diff
@@ -43,8 +44,8 @@ def update(
         _logger.debug("Progress update: %s", f"{self}")
 
     @classmethod
-    def create(cls) -> "TaskProgress":
-        return cls.parse_obj({"message": "", "percent": 0.0})
+    def create(cls, task_id: TaskId | None = None) -> "TaskProgress":
+        return cls.parse_obj({"task_id": task_id, "message": "", "percent": 0.0})
```

Comment:
MINOR: here just `cls(task_id=task_id)` and leave the defaults by defintion

Reply:
I am sorry, but I don't get it. Maybe I need some more ☕  but I don't see exactly what you mean. Do you mean something like this?
```python
    def create(cls, task_id: TaskId | None = None) -> "TaskProgress":
        d = {"message": "", "percent": 0.0}
        if task_id is not None:
            d.update(task_id=task_id)
        return cls.parse_obj(d)
```

Reply:
`create` is basically a constructor with a single argument, right?  in C++ we would use overloading and create multiple constructors with different signature but in python we use an extra `classmethod`. Note that here
- `create` is a  `classmethod`
-  the class `TaskProgress` is a  pydantic `BaseModel`
- we call the constructor of `TaskProgress` using `cls`
- `message` and `percent` have already default values by definition

```python
    @classmethod
    def create(cls, task_id: TaskId | None = None) -> "TaskProgress":
            return cls(task_id=task_id)
 ```
 
 would work

Reply:
Ah, sorry OK I misunderstood. I thought you meant the class defaults should take care of the default value of `task_id` not the others. Sure, I will adapt to your suggestion, thanks.